src/Simulation/EntryPointDriver.Tests/Tests.fs

```diff
@@ -878,9 +882,77 @@ let ``Shows help text for submit command with default target`` () =
                       --output <FriendlyUri|Id>                           The information to show in the output after the job is submitted.
                       --dry-run                                           Validate the program and options, but do not submit to Azure Quantum.
                       --verbose                                           Show additional information about the submission.
-                      -n <n> (REQUIRED)                                   A number.
-                      --pauli <PauliI|PauliX|PauliY|PauliZ> (REQUIRED)    The name of a Pauli matrix.
-                      --my-cool-bool (REQUIRED)                           A neat bit.
                       -?, -h, --help                                      Show help and usage information"
     let given = testWithTarget "foo.target" "Help"
     given ["submit"; "--help"] |> yields message
+
+[<Fact>]
+let ``Supports simulating multiple entry points`` () =
```

Comment:
Is there also test coverage for multiple entry points in multiple namespaces?
Edit: Based on the Q# file it doesn't look like it. It would be good to add.

Reply:
I've updated the existing tests to include an entry point in a separate namespace.